modules/apache-airflow/templates/values.yaml

```diff
@@ -108,7 +108,9 @@ images:
     pullPolicy: IfNotPresent
 
 # Select certain nodes for airflow pods.
-nodeSelector: {}
+nodeSelector: {
+  topology.kubernetes.io/zone: us-east-1a
```

Comment:
Are not all types of airflow pods affected, therefore we just adjust to ondemand for the specific ones instead of here (it looks like this affects all airflow pods?)

Reply:
Not every type of airflow pod needs to be ran in an on-demand instance. For example the web UI can run on spot instances because if it goes down we just lose UI access for a short bit.

But if a worker or part of the infra of airflow goes down, it has a larger impact.

Reply:
Thanks for explaining!